chaosgcp/apphub/actions.py

```diff
@@ -0,0 +1,295 @@
+# Copyright 2024 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from typing import Dict, Any
+
+from chaoslib.types import Configuration, Secrets
+from chaosgcp.lb.actions import (
+    inject_traffic_faults,
+    remove_fault_injection_traffic_policy,
+)
+from google.cloud import compute_v1, apphub_v1
+from google.cloud import resourcemanager_v3
+
+
+__all__ = [
+    "inject_fault_if_url_map_exists_app_hub",
+    "remove_fault_if_url_map_exists_app_hub",
+]
+logger = logging.getLogger("chaostoolkit")
+
+
+def apphub_list_app_services(
+    host_project_id: str,
+    location: str,
+    application_id: str,
+    configuration: Configuration = None,
+    secrets: Secrets = None,
+) -> None:
```

Comment:
What's the purpose of this function as it doesn't return anything? Is this a private function? debug? a probe?

Reply:
it is removed in new pull request . It is no longer needed.